plugins/yandex_reverse_search.py

```diff
@@ -0,0 +1,67 @@
+# Userge Plugin for Labstack Uploads (https://up.labstack.com)
+# Author: Sumanjay (https://github.com/cyberboysumanjay) (@cyberboysumanjay)
+# All rights reserved.
+
+import os
+import string
+import random
+
+import requests
+
+from userge import userge, Config, Message
+from userge.utils import progress
+
+
+@userge.on_cmd("yrs", about={
+    'header': "Yandex Reverse Search",
+    'description': "Reverse Search any Image/sticker",
+    'usage': "{tr}yrs [Reply to image | sticker]",
+    'note': "Gif & Animated Stickers won't work!"})
+async def labstack(message: Message):
+
+    await message.edit("Found Yandex Result. Lemme pass some Soup!")
+    if not os.path.isdir(Config.DOWN_PATH):
+        os.mkdir(Config.DOWN_PATH)
+
+    dl_loc = None
+
+    if message.reply_to_message and message.reply_to_message.media:
+        dl_loc = await message.client.download_media(
+            message=message.reply_to_message,
+            file_name=Config.DOWN_PATH,
+            progress=progress,
+            progress_args=(message, "Found Yandex Result. Lemme pass some Soup!")
+        )
+    else:
+        return await message.err("Media not found!")
+
+    filesize = os.path.getsize(dl_loc)
+    filename = os.path.basename(dl_loc)
+
+    file = {"name": filename, "type": "", "size": int(filesize)}
+    user_id = ''.join(random.choice(
+        string.ascii_lowercase + string.ascii_uppercase + string.digits
+    ) for _ in range(16))
+    data = {"ttl": 604800, "files": [file]}
+    headers = {
+        'up-user-id': user_id,
+        'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64)'
+                      'AppleWebKit/537.36 (KHTML, like Gecko) '
+                      'Chrome/83.0.4103.97 Safari/537.36'
+    }
+    r = requests.post(
```

Comment:
`requests.post` is a synchronous call. it will block userbot for a while or until get a response. can you use `aiohttp` or our pool ?

https://github.com/UsergeTeam/Userge-Plugins/blob/a41d1586bf9b4d51e76b6116f73f220489c334fd/plugins/zippyshare.py#L10

https://github.com/UsergeTeam/Userge-Plugins/blob/a41d1586bf9b4d51e76b6116f73f220489c334fd/plugins/zippyshare.py#L45-L47

https://github.com/UsergeTeam/Userge-Plugins/blob/a41d1586bf9b4d51e76b6116f73f220489c334fd/plugins/zippyshare.py#L22